ui/page/wallets/privacy_page.go

```diff
@@ -295,26 +298,37 @@ func (pg *PrivacyPage) Handle() {
 
 	if pg.allowUnspendUnmixedAcct.Changed() {
 		if pg.allowUnspendUnmixedAcct.IsChecked() {
-			modal.NewInfoModal(pg.Load).
-				Title("Warning: Danger alert!").
-				Body("Are you sure you want to allow spending from unmixed accounts?").
-				NegativeButton(values.String(values.StrCancel), func() {
-					pg.allowUnspendUnmixedAcct.SetChecked(false)
-				}).
-				PositiveButtonStyle(pg.Theme.Color.Surface, pg.Theme.Color.Danger).
-				PositiveButton("Allow", func() {
+			textModal:= modal.NewTextInputModal(pg.Load).
+			SetTextWithTemplate(). 
+			Hint(""). 
+			PositiveButtonStyle(pg.Load.Theme.Color.Danger, pg.Load.Theme.Color.InvText). 
+			PositiveButton("Confirm", func(textInput string, tim *modal.TextInputModal) bool {
+				if textInput != "I understand the risks" {
+					pg.Toast.NotifyError("Please enter correct passphrase", 10)
+					tim.IsLoading = false
+				} else {
 					pg.wallet.SetBoolConfigValueForKey(dcrlibwallet.AccountMixerConfigSet, false)
-				}).
-				Show()
+					tim.Dismiss()
+				}
+				return tim.IsLoading
+			})
+
+			textModal.Title("Confirm to allow spending from unmixed accounts").
```

Comment:
Create title stringin values package and call within here
Here after this line:
https://github.com/planetdecred/godcr/blob/796ab852f937b39b90f579447099616cca7dd58b/ui/values/strings.go#L230

Reply:
Since the title is only being used once, I didn't think it needed to be declared in the strings package

Reply:
Yeah but most of the strings there are specific to different models and are used just once. It is just a way to keep things organized

Reply:
Alright, noted. Thanks.

Reply:
The values /strings.go file should not be modified at any point please. Kindly hard code your string variables.